### DISCUSSION TITLE
Log seems noisy in CI

### DISCUSSION THREAD

**Discussion by jgsuess** [oracle/macaron#343] [Q&A] [answered]
I find that the default setting for Macaron talks a lot about its actual achievements in `info` level. This clutters the shell a lot. Is there a way to reduce the log level or redirect the level?

**Comment by jgsuess** [comment]
For example, this:
`
2023-06-28 02:08:04,555 [INFO] Didn't find any target run of 6065946 on page 1.
`
Does an operator of the container really need to know?

**Reply by behnazh-w** [reply, marked as answer]
Thanks for the feedback. We already have a ticket for reducing the info-level logs: https://github.com/oracle/macaron/issues/229
